io/include/traccc/io/csv/cell.hpp

```diff
@@ -30,4 +33,14 @@ struct cell {
                    timestamp, value);
 };
 
+inline traccc::cell make_cell_edm(const traccc::io::csv::cell& csv_cell) {
+
+    traccc::cell ce;
+    ce.channel0 = csv_cell.channel0;
+    ce.channel1 = csv_cell.channel1;
+    ce.activation = csv_cell.value;
+    ce.time = csv_cell.timestamp;
+    return ce;
+}
```

Comment:
This should not be in this header file. :confused: You use this function only in the `traccc::io` library's implementation. So the function not only shouldn't be implemented `inline` in the header, it shouldn't even be declared in a public header. :thinking:

Reply:
Made a header and source for make_measurement_edm